joomla/joomla-cms
# Show Redirect Status Code in Redirection List

This PR adds a new column to the redirection list if the advanced mode is active. In this new column you can see the redirect status code defined for each redirected URL.

Test instructions:
- Disable advanced mode (if enabled) => the redirection list consists of 8 columns, the redirect status code ist not shown.
- Enable advanced mode => the redirection list consists of 9 columns, the redirect status code is shown in a column between the 404 Hits and the ID.
